### PR TITLE
TeamCity: Add extension function and refactor ETK build

### PR DESCRIPTION
**Pull request by noahtallen**
#### Changes proposed in this Pull Request
- refactors the ETK build into a separate file
- adds an extension function for common bash script properties.
- Adds phpcs to ETK

#### Testing instructions
- Check build output for ETK TC task and make it passes, and that the lint check passes.

### CODE REVIEW

**Comment by noahtallen** [comment]
works great:

Screen Shot 2021-02-26 at 12 52 11 PM

(also note that phpcbf runs as a precomit hook, so this error is contrived)

File: .teamcity/settings.kts [hunk 1]
```diff
@@ -1366,6 +1366,7 @@ object WPComPlugins_EditorToolKit : BuildType({
 
 				cd apps/editing-toolkit
 				yarn test:js --reporters=default --reporters=jest-junit --maxWorkers=${'$'}JEST_MAX_WORKERS
+				yarn lint:php
```
Comment (scinos) [inline .teamcity/settings.kts]: I suggest moving it to a separate step (i.e. a new `script` block): the overhead is minimal and we'll get more info about which step failed and its average duration.

**Comment by noahtallen** [comment]
@scinos this should be ready for review now 👍

**Comment by scinos** [comment]
Nice!!

**Review by scinos — APPROVED** [review]

**Pull request merged**